feature-detects/mediastream.js

```diff
@@ -0,0 +1,21 @@
+/*!
+{
+  "name": "MediaRecorder API",
+  "property": "mediarecorder",
+  "caniuse": "mediarecorder",
+  "tags": ["mediarecorder"],
+  "authors": ["Jaikishan Brijwani"],
+  "notes": [{
+    "name": "MDN Docs",
+    "href": "https://developer.mozilla.org/en-US/docs/Web/API/MediaStream_Recording_API"
+  }]
+}
+!*/
+/* DOC
+Detects support for the MediaStream Recording API for detecting mediastreaming.
+*/
+
+
+define(['Modernizr'], function (Modernizr) {
+    Modernizr.addTest('mediarecorder', typeof MediaRecorder !== undefined );
```

Comment:
I actually did some work with [BrowserStack](https://browserstack.com/) (because Firefox 26 doesn't load the test page) and found that it did not work in Safari (one of the few browsers supported media stream)
![Console](https://user-images.githubusercontent.com/45839898/95027517-6b9ac480-0699-11eb-8b32-5aa3c98d9f99.PNG)

It was just that the `undefined` needed to be inside quotation marks. I give the answer as a suggestion (you should be able to click to apply the changes)

```suggestion
    Modernizr.addTest('mediarecorder', typeof MediaRecorder !== "undefined");
```